gui/packages/desktop/src/main/daemon-rpc.js

```diff
@@ -638,3 +638,28 @@ function transformObjectKeys(object: Object, keyTransformer: (string) => string)
   }
   return object;
 }
+
+export function defaultSettings(): Settings {
```

Comment:
FYI convenience methods to produce blank settings and tunnel state, used as initial values in renderer and main to avoid dealing with nullability.